examples/vector_search.ipynb

```diff
@@ -46,7 +46,7 @@
    "outputs": [],
    "source": [
     "!pip install superduperdb\n",
-    "!pip install ipython openai==1.1.2"
+    "!pip install ipython openai==1.1.2 numpy==1.24.4"
```

Comment:
It worked without this for me. I just needed to restart the kernel after installation.

Reply:
Strange. Can you test the MNIST notebook?

Reply:
We can just add this `code block` like this instead of the extra numpy dependency. The reason for this is that colab comes with many dependencies including numpy. When we install a package, it installs many packages which are already installed by colab. A clash occurs most of the time because of the versioning. So after installing large packages, I always run this as a safety net. 

<img width="284" alt="image" src="https://github.com/SuperDuperDB/superduperdb/assets/26186000/4ccdd589-5c85-4278-9978-ca7ca8069e32">